GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using UnityEditor;
+using UnityEngine;
+using UnityEditor.SceneManagement;
+using UnityEngine.UI;
+using System.IO;
```

Comment:
The statements aren't consistently order. Run the IDE's "optimize imports" feature on all files.

Reply:
Resolved. Additionally fixed the optimize imports tool issue as well.